src/GT-Spotter-UI/GTSpotterHeaderBrick.class.st

```diff
@@ -53,12 +52,7 @@ GTSpotterHeaderBrick >> helpButton: aBrick [
 	
 	helpButton
 		hintTitle: 'Spotter help';
-		action: [ 
-			self spotterModel exitDueTo: #actionButton.
-			tree := HelpBrowser open instVarNamed: #treeMorph.
-			item := tree allItems detect: [:each | 
-				each complexContents item key = GTSpotterHelp key].
-			tree selection: item ].
+		action: [ self inform: 'GT Help is not available anymore' ].
```

Comment:
Why keeping a button that does nothing but showing that something is not available? Isn't it better to just remove the button from the UI?

Reply:
yeah, I agree with this, @astares :)